tests/sdks/python/containers/lowkey_vault_container.py

```
@@ -0,0 +1,101 @@
+    def create_provider(self) -> AzureKeyVaultSecretProvider:
+        return AzureKeyVaultSecretProvider(self.create_secret_client())
+
+    def _wait_until_ready(
+        self,
+        max_retries: int = 30,
+        delay: float = 1.0,
+    ) -> None:
+        url = f"{self._vault_url}/ping"
+        for attempt in range(max_retries):
+            try:
+                response = requests.get(url, timeout=2, verify=False)
```

Ruff (0.15.10)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)